mflowgen/Tile_MemCore/custom-flowgen-setup/setup.tcl

```diff
@@ -0,0 +1,35 @@
+set active_scenarios [list UNIFIED_BUFFER FIFO SRAM]
```

Comment:
We should port this to the pe tile flow as well, so we don't have to override the entire node therel. That can be in a future PR though.

Reply:
Yeah was planning to update the PE flow after this.